internal/backends/winit/winitwindowadapter.rs

```diff
@@ -283,6 +283,31 @@ impl WindowAdapter for WinitWindowAdapter {
     fn window(&self) -> &corelib::api::Window {
         self.window.get().unwrap()
     }
+
+    fn renderer(&self) -> &dyn i_slint_core::renderer::Renderer {
```

Comment:
unfortunately, even if Renderer is sealed and not re-exported by the public crate, one will still be able to call function on them in their code.

```
window_adapter.renderer().text_size(Default::default(), "abc", Default::default(), Default::default())
```